inductiva/simulation/command.py

```diff
@@ -0,0 +1,19 @@
+"""Class for construction of simulator commands."""
+
+
+class Command():
+    """Base class for construction of simulator commands."""
+
+    def __init__(self, **kwargs):
+        """Command constructor.
+        Args:
+            method_name: The name of the method to be executed in 
+            the simulation API.
+            **kwargs: Additional keyword arguments to be passed to
+            the simulation API method."""
+
+        self.kwargs = kwargs
+
+    def get_args(self):
+        """Return the command as a dictionary."""
```

Comment:
I think this is returning the command *flags* as a dictionary. Also, if `self.kwargs` stores the flags, why not call it explicitly `self.flags`?